_posts/2016-07-21-The-Ready-Player-One-Approach-To-Defense.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: The Ready Player One Approach to Defense
+date: 21-07-2016
+categories: vanity security thinks sci-fi
+---
+# The Ready Player One Approach to Defense
+
+> What if computer emulation wasn't just about nostalgia? What if we could leverage the power of being olde and having knowledge of defunct hardware and software and legitimately put it to use today?
+
+![Rad!]({{site.url}}assets/images/retro-console.gif)
```

Comment:
It's not very portable though is it. I like that the repository is entirely self contained but I also don't like having a jekyll link in the markdown and would prefer to have a global URL. I'm honestly open to suggestion.